pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+python_files = Test*.py
```

Comment:
Should we include lower case here as well?

Reply:
do we have both? (and if so, why ;)

Reply:
Probably laziness/cross-platform issues early on.
I was originally thinking better to catch accidental mistakes, but I think you're probably right, better to not let them proliferate.  I withdraw my comments :)